docs/src/configuring-zed.md

```diff
@@ -389,7 +389,8 @@ List of `string` values
 ```json
 "tabs": {
   "close_position": "right",
-  "git_status": false
+  "git_status": false,
+  "file_icons": false,
```

Comment:
The value here should be true right because its default value example.

Reply:
Ah yes! Good catch, thank you!